### PR TITLE
Feature/add tests for helper methods 39

### PR DESCRIPTION
**Pull request by mathieulemieux**
I added additionnal tests to cover all scenarios.

### CODE REVIEW

File: test/graphkb.test.js [hunk 1]
```diff
@@ -1,4 +1,108 @@
...
+    test('use source rank to sort results', () => {
+        expect(orderPreferredOntologyTerms(
+            { source: { sort: 1 }, sourceId: 1 },
+            { source: { sort: 2 }, sourceId: 2 },
+        )).toBe(-1);
+        expect(orderPreferredOntologyTerms(
+            { source: { sort: 2 }, sourceId: 1 },
+            { source: { sort: 1 }, sourceId: 2 },
+        )).toBe(1);
+        expect(orderPreferredOntologyTerms(
```
Comment (creisle) [inline test/graphkb.test.js:81]: are these in the wrong test block? only the first 2 expects appear to have the sort property

Comment (mathieulemieux) [reply test/graphkb.test.js:81]: Not sure to understand the question.
Lines 73-80 of ./test/graphkb.test.js are intended to test lines 181-184 of ./src/graphkb.js

Comment (creisle) [reply test/graphkb.test.js:81]: Not a blocker, I just meant that the title 'use source rank to sort results' implies testing the rank/sort field of source so you might want the other tests in a different test block with a more descriptive name. Or remove "rank" from the name.

**Review by creisle — APPROVED** [review]

File: test/graphkb.test.js (hunk 1 above)
Comment (mathieulemieux) [reply test/graphkb.test.js:81]: Ok, I see. I just blindly follow the [code](https://github.com/bcgsc/pori_graphkb_loader/blob/007cab68a87f84c47a38c98452dd0b67a56ec3ad/src/graphkb.js#L180)  structure without questionning (Look, I used your trick!)

**Pull request merged**